src/recon_buildblock/BinNormalisation.cxx

```diff
@@ -145,7 +145,7 @@ apply(ProjData& proj_data,
   if (is_null_ptr(symmetries_sptr))
     symmetries_sptr.reset(new TrivialDataSymmetriesForBins(proj_data.get_proj_data_info_sptr()->create_shared_clone()));
 
-  const std::vector<ViewSegmentNumbers> vs_nums_to_process = 
+  const std::vector<ViewSegmentTOFNumbers> vs_nums_to_process =
     detail::find_basic_vs_nums_in_subset(*proj_data.get_proj_data_info_sptr(), *symmetries_sptr,
                                          proj_data.get_min_segment_num(), proj_data.get_max_segment_num(),
                                          0, 1/*subset_num, num_subsets*/);
```

Comment:
Check this usage for `timing_pos_num` etc? Is this correct?

Reply:
not sure what you mean here, but in most cases, `BinNormalisation` objects won't have a `timing_pos` index (but they could)

Reply:
I'll check the current implementation to see what it needs.